met-web/src/locales/en/gdx.json

```diff
@@ -309,5 +309,26 @@
             "The page you're looking for cannot be found.",
             "The page you're looking for might have been removed, moved or is temporarily unavailable."
         ]
+    },
+    "NoResults": {
```

Comment:
I think until we work on the ticket DESENG-611 which is to have the static text based on language instead of tenant we will have to maintain in both places else it will not display the text in GDX tenant.

Reply:
Ah okay so what I understand from this is that there's no way to mix locale files. In other words, if there's a gdx.json locale file, none of the default.json locale strings can be used. Thus, we need to duplicate every string for now, is that right?

Reply:
Yes that's correct.